src/coord/config.c

```diff
@@ -12,6 +12,7 @@
 #include "rmutil/util.h"
 #include "rmutil/strings.h"
 #include "hiredis/hiredis.h"
+#include "../module.h"
```

Comment:
This sort of include can be a bit problematic, are we sure it is needed?
If so then you can mark this as resolved.